source/dataman/AMDbUpgrade1Pt6.cpp

```diff
@@ -0,0 +1,141 @@
+#include "AMDbUpgrade1Pt6.h"
+
+#include "util/AMErrorMonitor.h"
+
+#include <QStringBuilder>
+
+AMDbUpgrade1Pt6::AMDbUpgrade1Pt6(const QString &databaseNameToUpgrade, QObject *parent)
+	: AMDbUpgrade(databaseNameToUpgrade, parent)
+{
+
+}
+
+AMDbUpgrade1Pt6::~AMDbUpgrade1Pt6()
+{
+
+}
+
+QStringList AMDbUpgrade1Pt6::upgradeFromTags() const
+{
+	return QStringList();
+}
+
+bool AMDbUpgrade1Pt6::upgradeNecessary() const
+{
+	return databaseToUpgrade_->tableExists("VESPERSXRFScanConfiguration_table")
```

Comment:
Are the old tables removed at the end of the upgrade? If they aren't, it looks like this database upgrade will happen every time on startup for existing applications.